apps/condo/domains/common/utils/sberCloudFileAdapter.js

```diff
@@ -87,7 +87,7 @@ class SberCloudFileAdapter {
     save ({ stream, filename, id, mimetype, encoding, meta = {} }) {
         const fileData = {
             id,
-            originalFilename: filename,
+            originalFilename: Buffer.from(filename, 'latin1').toString('utf8'),
```

Comment:
How do you know, that incoming filename had `latin1` encoding?

Reply:
You can use chardet as we already have it in project